relay/auth/hmac/store.go

```diff
@@ -0,0 +1,34 @@
+package hmac
+
+import (
+	"sync"
+
+	log "github.com/sirupsen/logrus"
+)
+
+// TokenStore is a simple in-memory store for token
+// With this can update the token in thread safe way
+type TokenStore struct {
```

Comment:
This should probably live in a different package than hmac (together with `Token`) since it's accessed outside the hmac package

Reply:
The problem with it the unmarshal function. It is not exported. In the store, we marshal the token  but in the hmac pkg unmarshal it. So the hmac will depend from the token and the token will depend from the hmac. Circle dependency

Reply:
I will do something in a separated PR